### PR TITLE
update handpose

### PR DESCRIPTION
**Pull request by fengwuyao**
Upgrade TFJS to 4.10.0

### CODE REVIEW

**Review by Linchenn — APPROVED** [review]

File: handpose/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "rollup-plugin-visualizer": "~3.3.2",
     "ts-node": "~8.8.2",
     "tslint": "~5.18.0",
-    "typescript": "~3.5.3",
+    "typescript": "~4.4.4",
```
Comment (mattsoulanille) [inline handpose/package.json:37]: Can we upgrade to 5, or does that cause type errors?

File: handpose/demo/package.json [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
   "dependencies": {
     "@tensorflow-models/handpose": "file:../dist",
-    "@tensorflow/tfjs-core": "^3.3.0",
-    "@tensorflow/tfjs-converter": "^3.3.0",
-    "@tensorflow/tfjs-backend-webgl": "^3.3.0",
-    "@tensorflow/tfjs-backend-wasm": "^3.3.0"
+    "@tensorflow/tfjs-core": "^4.9.0",
+    "@tensorflow/tfjs-converter": "^4.9.0",
+    "@tensorflow/tfjs-backend-webgl": "^4.9.0",
+    "@tensorflow/tfjs-backend-wasm": "^4.9.0"
```
Comment (mattsoulanille) [inline handpose/demo/package.json:16]: I think we can make this 4.10.0

File: handpose/package.json [hunk 3]
```diff
@@ -13,18 +13,18 @@
...
     "@rollup/plugin-node-resolve": "^7.1.1",
     "@rollup/plugin-typescript": "^3.0.0",
-    "@tensorflow/tfjs-converter": "^3.0.0",
-    "@tensorflow/tfjs-core": "^3.0.0",
-    "@tensorflow/tfjs-backend-cpu": "^3.0.0",
-    "@tensorflow/tfjs-backend-webgl": "^3.0.0",
+    "@tensorflow/tfjs-backend-cpu": "^4.9.0",
+    "@tensorflow/tfjs-backend-webgl": "^4.9.0",
+    "@tensorflow/tfjs-converter": "^4.9.0",
+    "@tensorflow/tfjs-core": "^4.9.0",
```
Comment (mattsoulanille) [inline handpose/package.json:27]: 4.10.0

File: handpose/package.json [hunk 4]
```diff
@@ -13,18 +13,18 @@
     "url": "https://github.com/tensorflow/tfjs-models.git"
   },
   "peerDependencies": {
-    "@tensorflow/tfjs-converter": "^3.0.0",
-    "@tensorflow/tfjs-core": "^3.0.0",
-    "@tensorflow/tfjs-backend-webgl": "^3.0.0"
+    "@tensorflow/tfjs-backend-webgl": "^4.9.0",
+    "@tensorflow/tfjs-converter": "^4.9.0",
+    "@tensorflow/tfjs-core": "^4.9.0"
```
Comment (mattsoulanille) [inline handpose/package.json:18]: 4.10.0

**Review by mattsoulanille — APPROVED** [review]
LGTM with nits. Thanks!

**Pull request merged**